packages/block-editor/src/components/link-control/search-item.js

```diff
@@ -46,7 +46,11 @@ export const LinkControlSearchItem = ( {
 					aria-hidden={ ! isURL }
 					className="block-editor-link-control__search-item-info"
 				>
-					{ ! isURL && ( safeDecodeURI( suggestion.url ) || '' ) }
+					{ ! isURL &&
+						( filterURLForDisplay(
+							safeDecodeURI( suggestion.url )
+						) ||
+							'' ) }
```

Comment:
`filterURLForDisplay()` also strips off `www` in addition to protocols. I'm not sure that's what we want here.

Reply:
It might not be what was prescribed by #19670, but if how we're presenting the link here aligns to the purpose of how a function like `filterURLForDisplay` is expected to be used, then we should embrace the consistency that comes with it. We might decide that trimming `www.` is not something we want to do for URLs filtered for display, but it's a change which should happen in the implementation of `filterURLForDisplay`, and not a decision we should want to make at any particular instance of where that function is used.